src/components/gas/GasSpeedLabelPagerItem.js

```diff
@@ -38,11 +49,11 @@ const transition = (
 );
 
 const GasSpeedLabelPagerItem = ({ label, selected, shouldAnimate }) => {
-  const ref = useRef();
+  const transitionRef = useRef();
 
   useEffect(() => {
-    if (shouldAnimate && ref.current) {
-      ref.current.animateNextTransition();
+    if (shouldAnimate && transitionRef.current) {
+      transitionRef.current.animateNextTransition();
```

Comment:
If you wanna go fancy
`shouldAnimate && transitionRef?. current?. animateNextTransition();`